jablib/src/test/java/org/jabref/logic/importer/fetcher/transformers/ScopusQueryTransformerTest.java

```diff
@@ -0,0 +1,92 @@
+package org.jabref.logic.importer.fetcher.transformers;
+
+import java.util.Optional;
+
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.search.query.SearchQueryNode;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ScopusQueryTransformerTest {
+
+    private ScopusQueryTransformer transformer;
+
+    @BeforeEach
+    void setUp() {
+        transformer = new ScopusQueryTransformer();
+    }
+
+    @Test
+    void simpleTermTransformsToTitleAbsKeyAuth() {
+        SearchQueryNode queryNode = new SearchQueryNode(Optional.empty(), "machine learning");
+        Optional<String> result = transformer.transformSearchQuery(queryNode);
+
+        assertTrue(result.isPresent());
+        assertEquals("TITLE-ABS-KEY-AUTH(\"machine learning\")", result.get());
```

Comment:
AI. I begin to hate it. AI does not know how to test Optionals correctl.y

Merge in one line. AI can do.